widget/button_test.go

```diff
@@ -312,3 +312,21 @@ func TestButton_ChangeTheme(t *testing.T) {
 		test.AssertImageMatches(t, "button/theme_changed.png", w.Canvas().Capture())
 	})
 }
+
+func TestButtonCompatImportance(t *testing.T) {
+	test.NewApp()
+	defer test.NewApp()
+	test.ApplyTheme(t, theme.LightTheme())
+
+	//test backward compatibility of widget.Importance
+	var imp widget.ButtonImportance = widget.HighImportance
+
+	btn := widget.NewButton("test", func() {})
+	btn.Importance = imp
+	btn.Refresh()
+
+	w := test.NewWindow(btn)
+	defer w.Close()
+
+	test.AssertImageMatches(t, "button/compat_importance.png", w.Canvas().Capture())
```

Comment:
I don't think you need any image for this if you just want to make sure that it compiles (please also start comments with a space and then a capital letter):
```suggestion
	// Test backward compatibility of widget.Importance
	var imp widget.ButtonImportance = widget.HighImportance

	btn := widget.NewButton("test", func() {})
	btn.Importance = imp
```
Remember to delete the corresponding image as well.